develop/toolkit/api-libraries/polkadot-js-sdk.md

```diff
@@ -1,4 +1,109 @@
 ---
 title: Polkadot.js SDK
-description: TODO
----
\ No newline at end of file
+description: Interact with Polkadot SDK-based chains easily using the Polkadot.js SDK. Query chain data, submit transactions, and more via JavaScript.
+---
+
+# Polkadot.js SDK
+
+## Introduction
+
+The Polkadot.js SDK uses JavaScript / Typescript to interact with Polkadot and Polkadot SDK-based chains. It allows you to query nodes, read chain state, and submit transactions through a dynamic, auto-generated API interface.
```

Comment:
It seems redundant, at least for me, to say polkadot + polkadot sdk-based chains since polkadot is already a polkadot-sdk-based chain. I'd remove it:

 ```suggestion
The Polkadot.js SDK uses JavaScript/Typescript to interact with Polkadot SDK-based chains. It allows you to query nodes, read chain state, and submit transactions through a dynamic, auto-generated API interface.
```